src/main/java/com/example/tournamentMaker/team/FootballPlayerRequestWithoutGivingTeamName.java

```diff
@@ -6,9 +6,9 @@
 
 @Getter
 @AllArgsConstructor
```

Comment:
czy to dobra adnotacja w tym wypadku ?

Reply:
jeżeli pola są finalne, to @AllArgs przecież jest potrzebny, nawet kompilator tego nie chce przepuścić